### ISSUE TITLE
CI: Don't build API docs if on fork

### ISSUE THREAD

**Issue by ewels** [nf-core/tools#263]
Now that Travis builds the API docs, tests fail on forks because the travis setup doesn't have permissions to push the files.

Would be good to add another condition to the build that checks the repo name.

**Comment by matq007** [comment]
I have something like this in fusion-report. Won't this help?

```
before_deploy:
  - pip install sphinx sphinx-rtd-theme m2r
  - cd docs && make html && touch build/.nojekyll
  - cp -r ${TRAVIS_BUILD_DIR}/docs/example ${TRAVIS_BUILD_DIR}/docs/build/html

deploy:
  - provider: pages
    skip_cleanup: true
    github_token: $GITHUB_TOKEN
    local_dir: docs/build/html
    on:
      branch: master
      python: 3.6
```

**Comment by ewels** [comment]
How does that limit which fork it runs on?

**Comment by matq007** [comment]
Travis should by default skip deploy script on a PR 😕

**Comment by ewels** [comment]
Ah ok, that's nice. But travis runs on my fork each time I push a commit... So presumably would still run there even with `deploy`...

I was thinking of just checking if the repo org is `nf-core`. Would probably be best done in your `deploy` structure above though - best of both worlds 👍

**Comment by ewels** [comment]
Fixed in #394
